### DISCUSSION TITLE
UrlTracker Logging Every Request

### DISCUSSION THREAD

**Discussion by Hokusmokus22** [Infocaster/UrlTracker#110] [Q&A]
Hello,

I started working on a site that is using Umbraco 7 with UrlTracker. Recently our Umbraco logs are being spammed with UrlTracker info:

INFO  InfoCaster.Umbraco.UrlTracker.Helpers.LoggingHelper - UrlTracker HttpModule | Incoming URL is: umbraco/backoffice/UmbracoApi/Dashboard/GetDashboard?section=settings
INFO  InfoCaster.Umbraco.UrlTracker.Helpers.LoggingHelper - UrlTracker HttpModule | Response statuscode is not 404, UrlTracker won't do anything
INFO  InfoCaster.Umbraco.UrlTracker.Helpers.LoggingHelper - UrlTracker HttpModule | EndRequest end

Because of this, daily logs are reach 2gb - 3gb per day. I tried disabling logging through an app setting in the web.config, but the logs continue to write. Anyone have any idea how I can turn these off?

**Comment by D-Inventor** [comment]
Hi @Hokusmokus22 !

I don't have much understanding of the URL Tracker for Umbraco v7, but I had a look at the readme of the source for v7 and I found this paragraph:

> urlTracker:enableLogging
> 
> boolean (false)
> 
> Set to true to enable logging debug information of the Url Tracker. Uses umbraco's built-in logging mechanism with LogType set to 'debug'.
> umbracoDebugMode needs to be enabled too.

I'm not sure for which version exactly this is, as the source was rather messy when I took over the project. I would assume that this works for at least the latest version for Umbraco v7
